server/src/main/java/se/fortnox/reactivewizard/server/RwServer.java

```diff
@@ -145,16 +136,22 @@ public static void registerShutdownDependency(Runnable blockShutdownUntil) {
         RwServer.blockShutdownUntil = blockShutdownUntil;
     }
 
-    static void shutdownHook(ServerConfig config, DisposableServer server, LoopResources loopResources, ConnectionCounter connectionCounter) {
-        LOG.info("Shutdown requested. Will wait up to {} seconds...", config.getShutdownTimeoutSeconds());
+    static void shutdownHook(ServerConfig config, DisposableServer server, ConnectionCounter connectionCounter) {
+        LOG.info("Shutdown requested. Waiting {} seconds before commencing.", SECONDS_WAIT_BEFORE_SHUTDOWN);
+        try {
+            Thread.sleep(SECONDS_WAIT_BEFORE_SHUTDOWN * 1000);
```

Comment:
since the code already is blocking this is okay we think and we are on our way to shutdown the service.